pom.xml

```diff
@@ -394,7 +394,7 @@
         <version.org.infinispan.protostream>4.4.1.Final</version.org.infinispan.protostream>
         <version.org.jasypt>1.9.3</version.org.jasypt>
         <version.org.javassist>3.23.2-GA</version.org.javassist>
-        <version.org.jberet>1.3.9.Final</version.org.jberet>
+        <version.org.jberet>1.3.10.Final</version.org.jberet>
```

Comment:
ee-9/feature-pack/pom.xml should also be changed to contain an upgrade of jberet-core from 2.0.0.Final to 2.0.1.Final.